scripts/generateProptypes.ts

```diff
@@ -211,6 +211,7 @@ async function generateProptypes(
   const unstyledPropsFile = unstyledFile.replace('.d.ts', '.types.ts');
 
   const propsFile = tsFile.replace(/(\.d\.ts|\.tsx|\.ts)/g, 'Props.ts');
```

Comment:
To reflect https://github.com/mui/material-ui/pull/37533#issuecomment-1645235243

```suggestion
  // TODO remove, should only have .types.ts
  const propsFile = tsFile.replace(/(\.d\.ts|\.tsx|\.ts)/g, 'Props.ts');
```